controller/controller.go

```diff
@@ -242,7 +242,11 @@ func filterARecords(endpoints []*endpoint.Endpoint) []string {
 func (c *Controller) ScheduleRunOnce(now time.Time) {
 	c.nextRunAtMux.Lock()
 	defer c.nextRunAtMux.Unlock()
-	c.nextRunAt = now.Add(c.MinEventSyncInterval)
+	// Shedule only if a reconciliation is not already planned
```

Comment:
```suggestion
	// schedule only if a reconciliation is not already planned
```

Reply:
@Raffo Thanks for the review!
I fixed the typo in the comment. 
Is there anything else I must do, to move this forward and have it shipped?
Thank you.